packages/storybook/src/documentation/pages/accessibility.stories.mdx

```diff
@@ -0,0 +1,29 @@
+import { XGrid } from '@material-ui/x-grid';
+
+<Meta title="1. Docs/12. Accessibility" component={XGrid} />
+
+# Accessibility
+
+Throughout our journey, accessibility has always been one of the core value of Material-UI components.
+Hence our data-grid has no less than 70 accessibility standards implemented in our first beta version.
```

Comment:
```suggestion
Hence the data-grid has no less than 70 accessibility standards implemented in our first beta version.
```

@mbrookes What about we start a card with all the guidelines we have for writing the documentation? I have created this card https://trello.com/c/1Q7hqXHu/2714-technical-documentation-guidelines. I think that it would be great to look at what Google, AWS, Microsoft, etc technical writers use. Could you enrich it? :)